proofs/eo/cpc/theories/Nfa.eo

```diff
@@ -0,0 +1,39 @@
+; Declares a new sort which will be used to represent the states and transitions
```

Comment:
I would move this to `programs/Nfa.eo`. Seems like files in `theories` directory contain constructs to define the theory-symbols, not definitions for programs.